doc_template/examples_root/examples/nb/visual_behavior_neuropixels_LFP_analysis.ipynb

```diff
@@ -0,0 +1,2481 @@
+{
```

Comment:
<p>Could you please use a seeded random number generator here instead? This way the notebook execution will be consistent from run to run. https://numpy.org/doc/stable/reference/random/generator.html</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/AllenSDK/pull/2640/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
To add some more info, we added a workflow to run notebooks when they get merged into the main branch, so that we can make sure it runs and also update it with the latest code. The notebook will change without a seed.

Reply:
Ok just added the seed before calling random.choice. Is this what you meant?

Reply:
Yep, looks good.